shared/stories/index.js

```diff
@@ -1,13 +1,24 @@
 /* eslint-disable import/no-extraneous-dependencies, import/no-unresolved, import/extensions */
 // @flow
-import {configure} from '@storybook/react'
+import React from 'react'
+import {configure, addDecorator} from '@storybook/react'
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider'
+import materialTheme from '../styles/material-theme.desktop'
 
 // Load css
 import '../desktop/renderer/style.css'
 
+// Load common-adapter stories
 // $FlowIssue
 const req = require.context('../common-adapters', true, /\.stories\.js$/)
 
+// Add material-ui dependency
+addDecorator(story => (
```

Comment:
some sub dependencies still rely on material-ui (like icon). So if any component happens to need that it'll fail. as a workaround we wrap all our stories in material. TODO this'll have to be different on mobile